### PR TITLE
Updates

### PR DESCRIPTION
**Pull request by jewbmx**
all tested and working.

### CODE REVIEW

**Comment by cache-sk** [comment]
Note - don't forget to bump plugin version in addon.xml.. :)

**Comment by jewbmx** [comment]
I leave that kinda stuff to the main dev.

**Review by jsergio123 — CHANGES_REQUESTED** [review]
you have errors in your PR

File: lib/resolveurl/plugins/tunepk.py [hunk 1]
```diff
@@ -49,6 +52,47 @@ def get_media_url(self, host, media_id):
...
+        
+    def get_url(self, host, media_id):
+        return self._default_get_url(host, media_id, template='https://tune.pk/video/{media_id}/')
+
+    @classmethod
+    def get_settings_xml(cls):
+        xml = super(cls, cls).get_settings_xml()
+        xml.append('<setting label="Video Quality" id="%s_quality" type="enum" values="High|Medium|Low" default="0" />' % (cls.__name__))
+        return xml
+                    sources.append((vids[key]['label'], vids[key]['file']))
```
Comment (jsergio123) [inline lib/resolveurl/plugins/tunepk.py]: issue here

File: lib/resolveurl/plugins/veoh.py [hunk 2]
```diff
@@ -29,20 +31,22 @@ def __init__(self):
         self.net = common.Net()
 
     def get_media_url(self, host, media_id):
-        html = self.net.http_GET("http://www.veoh.com/iphone/views/watch.php?id=" + media_id + "&__async=true&__source=waBrowse").content
-        if not re.search('This video is not available on mobile', html):
-            r = re.compile("watchNow\('(.+?)'").findall(html)
-            if (len(r) > 0):
-                return r[0]
+        _json = self.net.http_GET("https://www.veoh.com/watch/getVideo/" + media_id).content
+        _data = json.loads(_json)
```
Comment (jsergio123) [inline lib/resolveurl/plugins/veoh.py]: wrap in try/else please

**Review by jsergio123 — CHANGES_REQUESTED** [review]

**Comment by jewbmx** [comment]
Both those are cache-sk's fixes, think both are exact copies from the pastebins

**Comment by cache-sk** [comment]
Sorry guys for making this mess, created separate pull #115

**Comment by jewbmx** [comment]
No worries, if the rest of my pr is good, jsergio can accept it first and then accept yours right afterwards to apply the rest of your fix.

**Pull request closed without merging**